src/modules/launcher/Wox.Infrastructure/StringMatcher.cs

```diff
@@ -212,8 +212,8 @@ private static int CalculateSearchScore(string query, string stringToCompare, in
             if (allSubstringsContainedInCompareString)
             {
                 int count = query.Count(c => !char.IsWhiteSpace(c));
-                int factor = count < 4 ? 10 : 5;
-                score += factor * count;
+                int threshold = 4;
+                score += count <= threshold ? 10 * count : threshold * 10 + (count - threshold) * 5;
```

Comment:
nit: @alekhyareddy I find this a bit hard to read.  Can we write this out longform?

Reply:
sounds good.